src/tests/multihost/ipa/test_subid_ranges.py

```diff
@@ -0,0 +1,110 @@
+        :id: 50bcdc28-00c8-11ec-bef4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1803943
+        :steps:
+            1. Test newuidmap command
+            2. Test newgidmap command
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        ssh1 = SSHClient(multihost.client[0].ip,
+                         username=user, password=test_password)
+        (result, result1, exit_status) = ssh1.exec_command(f"ipa  "
+                                                           f"subid-find"
+                                                           f"  --owner  "
+                                                           f"{user}")
+        user_details = result1.readlines()
+        uid_start = int(user_details[5].split(': ')[1].split('\n')[0])
+        uid_range = int(user_details[6].split(': ')[1].split('\n')[0])
+        gid_start = int(user_details[7].split(': ')[1].split('\n')[0])
+        gid_range = int(user_details[8].split(': ')[1].split('\n')[0])
+        (results1, results2, results3) = ssh1.exec_command("unshare"
+                                                           " -U bash"
+                                                           " -c 'echo $$"
+                                                           " > /tmp/unshare.pid; "
+                                                           "sleep 1000'")
+        time.sleep(2)
+        proces_id = int(execute_cmd(multihost,
+                                    "cat "
+                                    "/tmp/unshare.pid").stdout_text.strip())
+        uid = 0
+        count = 1
+        ssh1.exec_command(f"newuidmap {proces_id} {uid} {uid_start} {count}")
+        ssh1.exec_command(f"newgidmap {proces_id} {uid} {gid_start} {count}")
+        result = execute_cmd(multihost, f"cat /proc/{proces_id}/uid_map")
+        assert str(uid) in result.stdout_text
+        assert str(uid_start) in result.stdout_text
+        assert str(count) in result.stdout_text
+        result = execute_cmd(multihost, f"cat /proc/{proces_id}/gid_map")
+        assert str(uid) in result.stdout_text
+        assert str(gid_start) in result.stdout_text
+        assert str(count) in result.stdout_text
+        multihost.client[0].run_command(f'kill -9 {proces_id}')
+        ssh1.close()
+
+    def test_list_subid_ranges(self, multihost):
+        """
+        :Title: support subid ranges managed by FreeIPA
+        :id: 4ab33f84-00c8-11ec-ad91-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1803943
+        :steps:
+            1. Test list_subid_ranges command
+            2. Test list_subid_ranges -g command
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        ssh1 = SSHClient(multihost.client[0].ip,
+                         username=user, password=test_password)
+        (result, result1, exit_status) = ssh1.execute_cmd('kinit',
```

Comment:
What is the purpose of `kinit` (and `klist`)?

It is required in `subid_generate()` fixture to generate the range, but is it required for `ipa subid-find` here?
(It is not done in `test_subid_feature()`)

Reply:
klist can be removed . And kinit done once will work for all the tests

Reply:
https://github.com/SSSD/sssd/pull/5755/commits/4b831d2218a3a3bb85a5dfa5a249767b3b14d15a?file-filters%5B%5D=.py#r704510075